### DISCUSSION TITLE
haptic feedback

### DISCUSSION THREAD

**Discussion by ghoudi01** [GeekyAnts/NativeBase#5143] [Feature request]
### Description

we can add haptic feedback to BUTTON component as props

### Problem Statement

installing react-native-haptic-feedback and usgin there functionalitys in evry onPress method is difficult. 

### Proposed Solution or API

using `react-native-haptic-feedback`

`<Button haptic=“impactLight”>
<Text>button with haptic feedback</Text>
</Button>`

**Comment by Krithikj20** [comment]
Hi @ghoudi01, Thank you for reaching out to us. We will look into it.

**Reply by ghoudi01** [reply]
can i work on it
